doc/api/modules.md

```diff
@@ -342,7 +342,7 @@ If the given path does not exist, `require()` will throw an [`Error`][] with its
 <!--type=misc-->
 
 It is convenient to organize programs and libraries into self-contained
-directories, and then provide a single entry point to that library.
+directories, and then provide a single entry point to those directories.
```

Comment:
I would delete this sentence entirely. The next sentence is sufficient (although consider changing `folder` to `directory` in that sentence). This sentence mostly just gets in the way.

Reply:
Hi.
sorry for the late response and thanks for the commit. I agree that the suggested version is better.
What should I do now?
Should I now to push the changes to the fork and synchronize this Pull Request with other changes that have landed in master by following instructions given at https://github.com/nodejs/node/blob/master/doc/guides/contributing/pull-requests.md#setting-up-your-local-environment?
I'm pretty new to GitHub.

Reply:
> Should I now push the changes to the fork and synchronize this Pull Request with other changes that have landed in master by following instructions given at https://github.com/nodejs/node/blob/master/doc/guides/contributing/pull-requests.md#setting-up-your-local-environment, right?

Yes, I believe so. If you run into a problem, let me know. (We can try to walk you through it, or just do it for you.)

Reply:
Hi again.
I've tried to do what I described above ("...synchronize this Pull Request with other changes that have landed in master...").
Hope, it worked.
If it didn't work and you can "just do it for you.", please do it )